scripts/gz10/README.md

```diff
@@ -0,0 +1,50 @@
+# GZ10 Galaxy Catalog Dataset
+
+The GZ10 catalog, provided by Leung et al. (2018), features 17,736 galaxies with labels from the Galaxy Zoo 2 project. 
```

Comment:
Please cite the original labels as well (Willet 2013 if you mean the GZ2 version, though I'm not sure if that's the version you mean)